docs/docs/error-codes.md

```diff
@@ -0,0 +1,15 @@
+---
+id: error-codes
+title: React Error Codes
```

Comment:
I suggested "Error Decoder" and still like that (I think it's cute) but if you prefer this let's just do "Error Codes" without the "React".